packages/sonarwhal/src/lib/types/schemavalidationresult.ts

```diff
@@ -0,0 +1,8 @@
+import * as ajv from 'ajv';
```

Comment:
For consistency with everything else, can we rename this file to `schema-validation-result.ts`?

Reply:
sure thing!